backend/src/main/java/com/stampcrush/backend/application/manager/owner/util/PasswordEncryptor.java

```diff
@@ -0,0 +1,20 @@
+package com.stampcrush.backend.application.manager.owner.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+
+public class PasswordEncryptor {
+
+    private static final String ALGORITHM = "SHA-256";
```

Comment:
Can we expose our hash algorithm category?

i think we should protect our hash algorithm by using value annotation 

how do you think?